### ISSUE TITLE
Setup unit testing

### ISSUE THREAD

**Issue by shaunsaker** [AuxStudio/react-native-boiler-plate#8]
Write tests for the utils and anything else that is testable.

**Comment by shaunsaker** [comment]
Use [Jest](https://facebook.github.io/jest/).

Mock the [store](https://github.com/arnaudbenard/redux-mock-store).

**Comment by shaunsaker** [comment]
Add testing guide - where to store tests, cheatsheet e.g.

```js
jest.mock('react-native-simple-animators', () => ({
  AnimateTranslateY: 'AnimateTranslateY',
}));
```

**Comment by shaunsaker** [comment]
Closed by #24
